src/server/views/layout.njk

```diff
@@ -90,6 +87,11 @@
 {% endblock %}
 
 {% block bodyEnd %}
+  <script type="text/javascript" src="/assets/govuk-frontend/all.js"></script>
```

Comment:
1. I moved this down so it becomes a non-blocking load. Generally you would want to put client js down here anyway so you can be sure the document has loaded, incase the js initialises too quickly 
2. We were loading govuk-frontend scripts, but not initialising them, and then writing our own custom radio reveal code, but this is already included in govuk frontend

https://frontend.design-system.service.gov.uk/importing-css-assets-and-javascript/#if-your-javascript-is-not-working-properly